src/Koffee/KeyBinding.fs

```diff
@@ -24,6 +24,7 @@ let defaultsAsString = [
     ("h", OpenParent)
     ("H", Back)
     ("L", Forward)
+    ("gh", StartPrompt ShowHistory)
```

Comment:
I couldn't get the `Back`/`Forward` and `StartPrompt ShowHistory` to show up next to each other. Any tips, or just leave it as-is? @mattstermiller 

![image](https://user-images.githubusercontent.com/2477952/97669264-0497e200-1a84-11eb-9367-f951643e0818.png)

Reply:
This Settings list pulls the commands from `MainEvents.Bindable`, so that's where the ordering comes from. Move the new command in that list.

Reply:
Oh doy, I just added the ShowHistory there. Thank you :)